webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -162,7 +162,7 @@ export default {
         this.originalBody = params.activityBody;
         this.activityId = params.activityId;
         this.spaceId = params.spaceId;
-        this.templateParams = params.activityParams || {};
+        this.templateParams = params.activityParams || params.templateParams || {};
```

Comment:
en mode edit we need this modification because if we edited activity before the loading of the preview link the activity can't updated so we need to load the existing template params